package.json

```
@@ -79,16 +87,16 @@
     "@types/jquery": "~3.3.22"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "6.* || 8.* || >= 10.*"
```

Just flagging this as a breaking change (again, totally fine with me).

Considering that we only have a month or two left in Node 6's support matrix, we should strongly consider making this:

```suggestion
    "node": "8.* || >= 10.*"
```

---

I agree. I was planning on this being a major bump. I'd prefer to just jump up to 8+ and do a single major bump rather than two majors in quick succession. 

I'll go ahead with this unless there's any objections.

---

No objections from me...